src/Surfnet/StepupSelfService/SamlStepupProviderBundle/Exception/UnknownProviderException.php

```diff
@@ -20,11 +22,11 @@
 
 class UnknownProviderException extends RuntimeException
 {
-    public static function create($unknownProvider, array $knownProviders)
+    public static function create($unknownProvider, array $knownProviders): static
     {
         return new static(sprintf(
             'Unknown Generic SAML Stepup Provider requested "%s", known providers: "%s"',
-            is_object($unknownProvider) ? '(object)' . get_class($unknownProvider) : $unknownProvider,
+            is_object($unknownProvider) ? '(object)' . $unknownProvider::class : $unknownProvider,
```

Comment:
This might be an introduction of a bug? Or did you intentionally change the logic for this exception message?